src/main/java/com/ericsson/ei/frontend/BackEndInformationController.java

```diff
@@ -16,35 +16,107 @@
 */
 package com.ericsson.ei.frontend;
 
+import com.ericsson.ei.frontend.model.BackEndInformation;
+import com.ericsson.ei.frontend.utils.BackEndInstancesUtils;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
 
 import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
 
-@RestController
-public interface BackEndInformationController {
+@Controller
+public class BackEndInformationController {
+
+    @Autowired
+    private BackEndInformation backEndInformation;
+
+    @Autowired
+    private BackEndInstancesUtils backEndInstancesUtils;
 
-    @CrossOrigin
     @RequestMapping(value = "/get-instances", method = RequestMethod.GET)
-    ResponseEntity<String> getInstances(Model model);
+    public ResponseEntity<String> getInstances(Model model) {
+        return new ResponseEntity<>(backEndInstancesUtils.getInstances().toString(), HttpStatus.OK);
+    }
 
-    @CrossOrigin
     @RequestMapping(value = "/switch-backend", method = RequestMethod.POST)
-    ResponseEntity<String> switchBackEndInstance(Model model, HttpServletRequest request);
+    public ResponseEntity<String> switchBackEndInstance(Model model, HttpServletRequest request) {
+        try {
+            String body = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
+            backEndInstancesUtils.setInstances(new JsonParser().parse(body).getAsJsonArray());
+            backEndInstancesUtils.writeIntoFile();
+            for (BackEndInformation backEndInformation : backEndInstancesUtils.getInformation()) {
+                if (backEndInformation.isActive()) {
+                    backEndInstancesUtils.setBackEndProperties(backEndInformation);
+                }
+            }
+            return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
Don't we need to JSON Content Header here in the response. When I looked at AJAX call earlier, it expects the JSON content header.
This comment applies to all Responses here.

Reply:
don't clearly understand what do you mean?

Reply:
From the error in the bug that was reported,, what I could find about that was some info about that the Header ContentType is missing and might be the root cause to the error.
So this might solve the error.

Ajax Call (JS) expect HEADER ContentType JSON,,, so ResponseEntity need to have something added:
HttpHeaders headers = new HttpHeaders();
        headers.setContentType(MediaType.APPLICATION_JSON);
        return new ResponseEntity<>(responseBody, headers, HttpStatus.valueOf(statusCode));

Reply:
ok, thanks will fix it

Reply:
Added